Formula/ctail.rb

```diff
@@ -12,15 +13,21 @@ class Ctail < Formula
     sha256 "dfab40d65950327c679bde97a335779526c58e99f5679f32f95e517a7249e332" => :mountain_lion
   end
 
-  conflicts_with "byobu", :because => "both install `ctail` binaries"
-
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "apr" => :build
+  depends_on "apr-util" => :build
+
+  conflicts_with "byobu", :because => "both install `ctail` binaries"
 
   def install
-    system "./configure", "--prefix=#{prefix}", "--disable-debug"
+    system "./configure", "--prefix=#{prefix}", "--disable-debug", "--with-apr=#{Formula["apr"].bin}", "--with-apr-util=#{Formula["apr-util"].bin}"
```

Comment:
Use `.opt_bin` for these and please wrap onto multiple lines so it's not longer than 80 characters.